### PR TITLE
Switch save/loads to JSON. Remove XML.

### PR DESCRIPTION
**Pull request by brian-brazil**
Depends on #330.

### CODE REVIEW

File: src/com/carolinarollergirls/scoreboard/json/AutoSaveJSONState.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+              } catch ( Exception e ) {
+                ScoreBoardManager.printMessage("Could not load auto-saved scoreboard XML file "+f.getPath()+" : "+e.getMessage());
+                e.printStackTrace();
+              }
+            }
+
+        return false;
+    }
+ 
+    public static void loadFile(ScoreBoard scoreBoard, File f) throws Exception{
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/json/AutoSaveJSONState.java]: This mostly duplicates code from `LoadJsonScoreBoard` - maybe that should be factored into `ScoreBaordJSONSertter`

File: config/crg.scoreboard.properties [hunk 2]
```diff
@@ -17,7 +17,6 @@
 
 # These are the servlets that are started.
 # Unless you know what you are doing, you should not modify this.
-com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.servlet.1 = com.carolinarollergirls.scoreboard.jetty.XmlScoreBoardServlet
 com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.servlet.2 = com.carolinarollergirls.scoreboard.jetty.ScoreBoardVersionServlet
 com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.servlet.5 = com.carolinarollergirls.scoreboard.jetty.MediaServlet
```
Comment (frank-weinberg) [inline config/crg.scoreboard.properties:21]: I guess at this point we can just hardcode the last remaining values and get rid of the properties file.

Comment (brian-brazil) [reply config/crg.scoreboard.properties:21]: I've been planning to do that for some time.

File: src/com/carolinarollergirls/scoreboard/json/AutoSaveJSONState.java (hunk 1 above)
Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/json/AutoSaveJSONState.java]: Done

**Pull request merged**